assets/diagram-js.css

```diff
@@ -521,7 +521,7 @@ marker.djs-dragger tspan {
   height: 100vh;
   top: 0;
   left: 0;
-  z-index: 200;
+  z-index: 2000;
```

Comment:
Why 2000 over 200 over 1377? :slightly_smiling_face: 

`z-index` is already 200 times higher than the base. Overrides are possible within applications.